manager/processors/duplicate_tmplvars.processor.php

```diff
@@ -12,49 +12,31 @@
 }
 
 // count duplicates
-$name = $modx->getDatabase()->getValue($modx->getDatabase()->select('name', $modx->getDatabase()->getFullTableName('site_tmplvars'), "id='{$id}'"));
-$count = $modx->getDatabase()->getRecordCount($modx->getDatabase()->select('name', $modx->getDatabase()->getFullTableName('site_tmplvars'), "name LIKE '{$name} {$_lang['duplicated_el_suffix']}%'"));
+$tmplvar = EvolutionCMS\Models\SiteTmplvar::findOrFail($id);
```

Comment:
Use eager loading for the **tmplvarAccess** and **tmplvarTemplate** relationship